samples/snippets/csharp/VS_Snippets_CLR_System/system.io.directoryinfo.enumeratedirectories/cs/enumaratedirectories.csproj

```
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Let's go ahead and update to .NET 6. (That's been released and it's an LTS release).
```suggestion
    <TargetFramework>net6.0</TargetFramework>
```